bin/test/cucumber.rb

```diff
@@ -44,21 +44,26 @@
 
     sim_version = RunLoop::Version.new("#{sim_major}.#{sim_minor}")
 
-    if ENV["JENKINS_HOME"]
+    if RunLoop::Environment.jenkins?
```

Comment:
This section is still incorrec.t

Please remove the Jenkins branching.  Delete the `if/else`.